community_tutorials_and_guides/blazingsql/bsql_vs_pyspark_netflow.ipynb

```diff
@@ -194,19 +194,19 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 4.11 ms, sys: 23 µs, total: 4.13 ms\n",
-      "Wall time: 3.32 ms\n"
+      "The history saving thread hit an unexpected error (OperationalError('database or disk is full')).History will not be written to the database.\n"
```

Comment:
If the code throws a memory/disk full error then please check if the code runs with a smaller dataset. If it does then I would suggest using the original dataset size and clearing the outputs

Reply:
@Nicholas-7, what was the resolution on this?